virt/virtual_machines/virt-manage-vmis.adoc

```diff
@@ -0,0 +1,14 @@
+[id="virt-manage-vmis.adoc"]
```

Comment:
@lmandavi this doesn't look like a correct id. I don't think this should have the file extension in it. Can you fix in a new PR?

This is a hard error to catch :) @ousleyp

Reply:
Sure, will update when it's done. Thanks.

Reply:
Update was completed. PR was merged.